tests/unit_tests/dft/source/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+# SPDX-License-Identifier: Apache-2.0
+#===============================================================================
+
+#Build object from all test sources
+set(DFT_SOURCES
+  tmp.cpp
```

Comment:
`tmp.cpp` should be removed when rebasing onto the tests in https://github.com/oneapi-src/oneMKL/pull/261.

Reply:
Done in [c7ecd6e](https://github.com/oneapi-src/oneMKL/pull/259/commits/c7ecd6e7f65f9a21e70e8313421da7d9e913c7e4).